apps/condo/domains/ticket/components/TicketId/TicketResidentFeatures.tsx

```diff
@@ -1,25 +1,103 @@
+import { BillingReceipt } from '@app/condo/domains/billing/utils/clientSchema'
+import { BankCardIcon } from '@app/condo/domains/common/components/icons/BankCardIcon'
 import { Ticket } from '@app/condo/schema'
 import { MobileIcon } from '@condo/domains/common/components/icons/MobileIcon'
 
 import { Tooltip } from '@condo/domains/common/components/Tooltip'
+import { useIntl } from '@condo/next/intl'
+import { Col, Row } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import dayjs from 'dayjs'
 import get from 'lodash/get'
 import React from 'react'
-import { useIntl } from '@condo/next/intl'
 
-interface TicketResidentFeaturesProps {
-    ticket: Ticket
+interface MobileAppInstalledIndicatorProps {
+    isContactHasMobileApp: boolean
 }
 
-export const TicketResidentFeatures: React.FC<TicketResidentFeaturesProps> = ({ ticket }) => {
+const MobileAppInstalledIndicator: React.FC<MobileAppInstalledIndicatorProps> = ({ isContactHasMobileApp }) => {
     const intl = useIntl()
     const MobileAppInstalledMessage = intl.formatMessage({ id: 'pages.condo.ticket.MobileAppInstalled' })
     const MobileAppNotInstalledMessage = intl.formatMessage({ id: 'pages.condo.ticket.MobileAppNotInstalled' })
 
-    const isContactHasMobileApp = !!get(ticket, 'client')
-
     return (
         <Tooltip title={isContactHasMobileApp ? MobileAppInstalledMessage : MobileAppNotInstalledMessage}>
             <MobileIcon active={isContactHasMobileApp} />
         </Tooltip>
     )
+}
+
+interface PaymentsAvailableIndicatorProps {
+    ticketOrganizationId: string
+}
+
+/**
+ Availability is determined by:
+ 1. at least 1 receipt for the current or previous period has been uploaded to the organization.
+ 2. If the number of receipts for the previous period is not equal to 0,
+ then the number of receipts for the current period must be greater than or equal to the number for the previous one.
+ */
+const getIsPaymentsInMobileAppAvailable = (receiptsInCurrentPeriod, receiptsInPreviousPeriod) => {
```

Comment:
probably it's easy to create something like `useIsReceiptsAvailable` which should include `BillingReceipt.useObject( ... )`

Reply:
and it's not enough to just have receipts. You also need a acquiring context

Reply:
there is a condition in the receipts request `context: { organization: { id: ticketOrganizationId } }`
if there is no context, then receipts are not fetched